pages/draft-05/[slug].page.tsx

```diff
@@ -22,6 +22,7 @@ export default function StaticMarkdownPage({
   frontmatter: any;
   content: any;
 }) {
+  const markdownFile = 'readme';
```

Comment:
Why not doing `  const markdownFile = '_index';` like you did in other slug pages? We should avoid hardcode the names in the slug pages.